bundles/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/SatelHandlerFactory.java

```diff
@@ -80,8 +80,6 @@ public boolean supportsThingType(ThingTypeUID thingTypeUID) {
         if (effectiveUID == null) {
             if (DEVICE_THING_TYPES_UIDS.contains(thingTypeUID)) {
                 effectiveUID = getDeviceUID(thingTypeUID, thingUID, configuration, bridgeUID);
-            } else if (VIRTUAL_THING_TYPES_UIDS.contains(thingTypeUID) && bridgeUID != null) {
-                effectiveUID = new ThingUID(thingTypeUID, bridgeUID.getId());
             }
```

Comment:
Why is this removed?

Reply:
Because this part makes specific ThingUID for these thing types, which is not valid anymore. Before, for virtual thing types, thing UID was made as `satel:<virt_thing_type_id>:<bridge_id>`, because only one thing of such thing type was required to be configured. This is not possible anymore, because valid thing UID is `satel:<virt_thing_type_id>:<bridge_id>:<thing_id>`. So I removed this part and when adding new thing of virtual thing type it gets random thing ID like other types do.

Reply:
Understood.